CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+Boost 1.77.0:
+
+* Fixed memory deallocation bug in `object`.
```

Comment:
@vinniefalco are you satisfied with this changelog entry? Also, see release notes entry below.